me.timschneeberger.GalaxyBudsClient.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: me.timschneeberger.GalaxyBudsClient
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: GalaxyBudsClient
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+finish-args:
+  - --socket=x11 # https://github.com/AvaloniaUI/Avalonia/issues/1243 😔
+  - --share=ipc
+  - --share=network
+  - --device=all
+  - --allow=bluetooth
+  - --system-talk-name=org.bluez.*
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=com.canonical.Unity.LauncherEntry
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.*
+  - --filesystem=xdg-download:ro # xdg-desktop-portal integration only available starting with Avalonia 11.0.0-rc1.1
+modules:
+  - name: GalaxyBudsClient
+    sources:
+      - type: git
+        url: https://github.com/ThePBone/GalaxyBudsClient
+        commit: bc7fa96c27626eb5263649786e210b7590ab4f9f
+      - sources-arm64.json
+      - sources-x64.json
+    buildsystem: simple
+    build-commands:
+      - '. /usr/lib/sdk/dotnet6/enable.sh ; [[ $(uname -m) == "x86_64" ]] && export ARCH="x64" || export ARCH="arm64" ; dotnet publish -r linux-$ARCH -c Release -p:PublishTrimmed=true --self-contained true --source ./nuget-sources GalaxyBudsClient/GalaxyBudsClient.csproj'
+      - '[[ $(uname -m) == "x86_64" ]] && export ARCH="x64" || export ARCH="arm64" ; cp -r --remove-destination /run/build/GalaxyBudsClient/GalaxyBudsClient/bin/Release/net6.0/linux-$ARCH/publish/ /app/bin/'
```

Comment:
you can use `$FLATPAK_ARCH`

Reply:
or `build-options` with `arch` specific `env`.

Reply:
Thanks, it's much cleaner now.